qiskit_experiments/framework/composite/parallel_experiment.py

```diff
@@ -70,95 +72,111 @@ def __init__(
         )
 
     def circuits(self):
+        return self._combined_circuits(device_layout=False)
+
+    def _transpiled_circuits(self) -> List[QuantumCircuit]:
+        circuits = self._combined_circuits(device_layout=True)
+
+        # We may need to re-schedule the combined circuits
+        # NOTE: In the future this should be replaced with a minimal
+        # pass manager that only needs to run relevent scheduling passes
+        scheduling_method = getattr(self.transpile_options, "scheduling_method", None)
+        if scheduling_method not in {None, "asap", "as_soon_as_possible"}:
+            circuits = transpile(circuits, self.backend, **self.transpile_options.__dict__)
+
+        return circuits
+
+    def _combined_circuits(self, device_layout: bool) -> List[QuantumCircuit]:
+        """Generate combined parallel circuits from transpiled subcircuits."""
+        if not device_layout:
+            # Num qubits will be computed from sub experiments
+            num_qubits = len(self.physical_qubits)
+        else:
+            # Work around for backend coupling map circuit inflation
+            if self.backend:
+                coupling_map = self.backend.configuration().coupling_map
+            else:
+                coupling_map = None
+            if coupling_map is not None:
+                num_qubits = 1 + np.max(coupling_map)
```

Comment:
I did this because the transpiler inflates circuits based on the coupling map, not the backends number of qubits, so this was to be consistent with the previous behavior.

If a backend has `coupling_map=None` the circuit will only be inflated to the largest qubit in the layed-out circuit (which is equivalent to the largest physical qubit in the experiment in the else case)

Reply:
Okey, but usually `np.max(coupling_map) + 1 == self.backend.configuration().num_qubits`. What would be the expected behavior

- device has 3 qubits
- coupling map is [[0, 1], [1, 0]], i.e. q2 is intentionally isolated qubit
- run T1 experiment for every qubits in parallel

If I understand correctly current implementation cannot handle this. Do you know why terra transpiler does such weird qubit number investigation?